homework_18/Orange.java

```diff
@@ -0,0 +1,5 @@
+public class Orange extends Fruit {
+    public Orange() {
+        super(1.5F);
```

Comment:
Why do you put a float here, if class `Fruit` has a field `weight` with a type `double`?